dbft.go

```diff
@@ -393,6 +393,7 @@ func (d *DBFT) onPrepareResponse(msg payload.ConsensusPayload) {
 		return
 	}
 
+	d.Logger.Info("received PrepareResponse", zap.Uint("validator", uint(msg.ValidatorIndex())))
```

Comment:
you can use `zap.Uint16()` without convertion

Reply:
```go
d.Logger.Info("received PrepareResponse",
    zap.Uint16("validator", msg.ValidatorIndex()))
```

Reply:
I have to confess, it was a blatant copy-paste from some other function there.